docs/CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ To contribute code to ODK Collect, you will need to open a [pull request](https:
 
         git push
 
-1. Once you have completed your code changes, verify that you have followed the [style guidelines](https://github.com/getodk/collect/blob/master/CONTRIBUTING.md#style-guidelines).
+1. Once you have completed your code changes, verify that you have followed the [style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md).
```

Comment:
Good spot! I think this can just use a relative link (`CODE-GUIDELINES.md`) directly to the file, however. GitHub works nicely with them - you'll see there's an example above with `WIDGETS.md`.